org.eclipse.SUMO.yaml

```
@@ -0,0 +1,198 @@
+      - --with-xcursor
+      - --with-xrender
+      - --with-xrandr
+      - --with-opengl
+      - --with-shape
+      - --with-xshm
+    sources:
+      - type: archive
+        url: http://fox-toolkit.org/ftp/fox-1.6.57.zip
+        sha256: 86ce8967094f03924e6801482dbe1de3a8de2edc41094fea8313c8524bc088a0
+        x-checker-data:
+          type: anitya
+          project-id: 11427
+          url-template: http://fox-toolkit.org/ftp/fox-$version.zip
+
+  - name: Eigen3
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.bz2
+        sha256: b4c198460eba6f28d34894e3a5710998818515104d6e74e5cc331ce31e46e626
+        x-checker-data:
+          type: anitya
+          project-id: 13751
+          stable-only: true
+          url-template: https://gitlab.com/libeigen/eigen/-/archive/$version/eigen-$version.tar.bz2
+    cleanup:
+      - '*'
+
+  - name: openscenegraph
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.6.5.tar.gz
+        sha256: aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12
+        x-checker-data:
+          type: anitya
+          project-id: 6848
+          stable-only: true
+          url-template: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-$version.tar.gz
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+      - /usr/lib/sdk/openjdk17/installjdk.sh
+
+      # Disable CHECK_OPTIONAL_LIBS, require network.
+  - name: SUMO
+    buildsystem: cmake-ninja
+    build-options:
+      env:
+        LIBRARY_PATH: "/app/lib:/usr/lib"
+        LD_LIBRARY_PATH: "/app/lib:/usr/lib"
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk17/bin
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSUMO_UTILS:BOOL=true
+      - -DFMI:BOOL=true
+      - -DCHECK_OPTIONAL_LIBS:BOOL=true
+      - -DENABLE_JAVA_BINDINGS=1
+      - -DJava_JAVA_EXECUTABLE=/app/jre/bin
+      - -DJava_JAR_EXECUTABLE=/usr/bin
+      - -DJava_JAVAC_EXECUTABLE=/usr/bin
+      - -DJava_JAVADOC_EXECUTABLE=/usr/bin
+      - -DJava_JAVAH_EXECUTABLE=/usr/bin
+      - -DJAVA_JVM_LIBRARY=/app/jre/lib
+      - -DJAVA_INCLUDE_PATH=/app/jdk/include
+      - -DJAVA_INCLUDE_PATH2=/app/jdk/include/linux
+      - -DJAVA_AWT_INCLUDE_PATH=/app/jdk/include
+    post-install:
+      - install -d -m 755 ${FLATPAK_DEST}/share/applications
+      - install -m 755 ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
+      - install -m 755 ${FLATPAK_DEST}/share/mime/application
+      - install -m 755 ${FLATPAK_DEST}/share/metainfo
+      - install -Dm0644 build/package/sumo.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
```

```suggestion
      - install -Dm0644  -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/ build/package/sumo.png
```

Because of the rename icon (which will fix the desktop file)